tutorial/Lab2/README.md

```diff
@@ -201,6 +208,13 @@ Enough talking about the theory. Now let's give it a go!
 
 1. View the guestbook as per [Lab1](../Lab1/README.md), using the updated port for the guestbook service.
```

Comment:
Might be worth changing to "View the guestbook" also here?

Reply:
resolved!